wallet/src/de/schildbach/wallet/Constants.java

```diff
@@ -66,7 +66,7 @@
         switch (BuildConfig.FLAVOR) {
             case "prod":
             case "beta": {
-                DNS_SEED = new String[]{"dnsseed.dash.org"};
+                DNS_SEED = new String[]{"dnsseed.dash.org", "dnsseed.dashdot.io"};
```

Comment:
Maybe also masternode.io?
https://github.com/dashpay/dash/blob/master/src/chainparams.cpp#L298

Reply:
Good question.  masternode.io hasn't been working for the past year from what I could tell.